docs/source/yelpsoa_configs.rst

```diff
@@ -544,6 +544,29 @@ Example Job
             cpus: .5
             mem: 100
 
+Default Monitoring
+^^^^^^^^^^^^^^^^^^
+
+For any monitoring fields not set in a Tron job's configuration, PaaSTA will
```

Comment:
You don't need to duplicate this. 

There is already

```monitoring: A dictionary of values that configure overrides for monitoring parameters that will take precedence over what is in [[monitoring.yaml]]. These are things like team, page, etc.```

Put this into the "common options" section and I think that would be OK

Reply:
I've moved stuff to the `monitoring.yaml` section.